packages/editor/src/components/block-list/style.scss

```diff
@@ -1,104 +1,35 @@
 .editor-block-list__layout .components-draggable__clone {
-	& > .editor-block-list__block > .editor-block-list__block-draggable {
-		background: $white; // @todo: ensure this works with themes that invert the color
-		box-shadow: $shadow-popover;
-
-		@include break-small {
-			left: -$block-parent-side-ui-clearance - $border-width;
-			right: -$block-parent-side-ui-clearance - $border-width;
-
-			.editor-block-list__layout & {
-				left: -$border-width;
-				right: -$border-width;
-			}
-		}
-	}
-
 	// Hide the Block UI when dragging the block
 	// This ensures the page scroll properly (no sticky elements)
 	.editor-block-contextual-toolbar,
-	.editor-block-mover,
 	.editor-block-settings-menu {
 		// I think important is fine here to avoid over complexing the selector
 		display: none !important;
 	}
 }
 
-.editor-block-list__layout .editor-block-list__block-draggable {
-	position: absolute;
-	top: 0;
-	right: 0;
-	bottom: 0;
-	left: 0;
-	z-index: z-index(".editor-block-list__block-draggable");
-
-	> .editor-block-list__block-draggable-inner {
-		position: absolute;
-		top: 0;
-		right: 0;
-		bottom: 0;
-		left: 0;
-		visibility: hidden;
-
-		// Use opacity to work in various editor styles.
-		background-color: $dark-opacity-light-200;
-
-		.is-dark-theme & {
-			background-color: $light-opacity-light-200;
+.editor-block-list__layout .editor-block-list__block.is-selected { // Needs specificity to override inherited styles.
```

Comment:
It looks like the changes here introduced a regression in Spotlight mode.

It should be fixed here https://github.com/WordPress/gutenberg/pull/9951